src/views/governance/views/proposal/components/ConfirmProposalOverview.tsx

```diff
@@ -106,13 +106,13 @@ const ConfirmProposalOverview = ({ tx, ...props }: Props) => {
   const setProposalEditing = useSetAtom(isProposalEditingAtom)
 
   return (
-    <Container variant="layout.sticky" p={0} {...props}>
+    <Container p={0} {...props}>
       <Box
         sx={{
           maxHeight: 'calc(100vh - 124px)',
           display: 'flex',
           flexDirection: 'column',
-          overflow: 'hidden',
+          overflow: 'auto',
```

Comment:
Pretty sure the overflow was because of something, try it out with different resolutions